cli/tests/pcluster/configure/test_pcluster_configure/test_vpc_automation_no_vpc_in_region_public/pcluster.config.ini

```diff
@@ -23,6 +23,7 @@ master_subnet_id = subnet-12345678
 
 [queue compute]
 enable_efa = false
+enable_efa_gdr = false
```

Comment:
Why are we adding these defauls to all the tests?

Reply:
On `queue` section `enable_efa_gdr` has no default value, since it can be inherited from `cluster`